qiskit_experiments/library/hamiltonian/heat_zx.py

```diff
@@ -0,0 +1,220 @@
+        state flip by the pi-pulse echo, while Z error is amplified outward
+        the X-Z plane in the same manner.
+        Measurement is projected onto Y-axis in this setup.
+        Because the echoed axis are anti-commute with other Pauli terms,
+        errors in other axes are cancelled out to reduce rotation in the interrogated axis.
+        This enables to selectively amplify the Hamiltonian dynamics in the specific axis.
+        Note that we have always nonzero X rotation imparted by the significant ZX term,
+        the error along Y and Z axis are skewed by the nonzero commutator term.
+        This yields slight mismatch in the estimated coefficients with the generator Hamiltonian,
+        however this matters less when the expected magnitude of the error is small.
+        On the other hand, the error in the X axis is straightforward
+        because this is commute with the ZX term of the generator.
+
+        .. parsed-literal::
+                             (xN)
+                             ░ ┌───────┐                            ░
+            q_0: ────────────░─┤0      ├────────────────────────────░─────────
+                 ┌─────────┐ ░ │  heat │┌────────────────┐┌───────┐ ░ ┌───┐┌─┐
+            q_1: ┤ Rα(π/2) ├─░─┤1      ├┤ Rx(-1.0*angle) ├┤ Rβ(π) ├─░─┤ γ ├┤M├
+                 └─────────┘ ░ └───────┘└────────────────┘└───────┘ ░ └───┘└╥┘
+            c: 1/═══════════════════════════════════════════════════════════╩═
+                                                                            0
+
+                             (xN)
+                    ┌───┐    ░ ┌───────┐                       ░
+            q_0: ───┤ X ├────░─┤0      ├───────────────────────░─────────
+                 ┌──┴───┴──┐ ░ │  heat │┌───────────┐┌───────┐ ░ ┌───┐┌─┐
+            q_1: ┤ Rα(π/2) ├─░─┤1      ├┤ Rx(angle) ├┤ Rβ(π) ├─░─┤ γ ├┤M├
+                 └─────────┘ ░ └───────┘└───────────┘└───────┘ ░ └───┘└╥┘
+            c: 1/══════════════════════════════════════════════════════╩═
+                                                                       0
+
+        ZX-HEAT experiments are performed with combination of two
+        error amplification experiments shown above, where :math:`\\alpha, \\beta, \\gamma`
+        depend on the interrogated error axis, namely,
+        (``X``, ``X``, ``I``), (``Y``, ``Y``, ``I``), (``Y``, ``Z``, ``Rx(-pi/2)``)
+        for amplifying X, Y, Z axis, respectively.
+        The circuit in middle is repeated by ``N`` times for the error amplification.
+
+    # section: note
+        The ``heat`` gate is a special gate to represent the entangler pulse sequence.
+        This gate is usually not provided by the backend, and thus user must provide
+        the pulse definition to run this experiment.
+        This pulse sequence should be pre-calibrated to roughly implement the
+        ZX(angle) evolution otherwise selective amplification doesn't work properly.
```

Comment:
```suggestion
        The ``heat`` gate represents the entangling pulse sequence.
        This gate is usually not provided by the backend, and users must thus provide
        the pulse schedule to run this experiment.
        This pulse sequence should be pre-calibrated to roughly implement the
        ZX(angle) evolution otherwise selective amplification doesn't work properly.
```
Perhaps also metion how this pulse-schedule is transpilled in.